OPCB/functions/missions/idap.sqf

```diff
@@ -29,12 +29,13 @@ _stations = [_village,_guard] call CHAB_fnc_artilerry;
 	_townName = text _village;
 	hint format ["%1 is currently under heavy artilerry fire. Please save the Village.",_townName];
 }, [_nearestCity], 1.5, true, true, "", "alive _target", 6, false, ""]] remoteexeccall ["addaction",0,true];
-missionNamespace setVariable ["artilerry_rem",4];
+
+ArtilleryRemaining = 4;
 
 waitUntil {
```

Comment:
A few seconds of `sleep` inside low priority `waitUntil`s like this can help with overall performance.